wagtail/admin/checks.py

```diff
@@ -189,7 +189,15 @@ def wagtail_admin_base_url_check(app_configs, **kwargs):
 def file_overwrite_check(app_configs, **kwargs):
     from django.conf import settings
```

Comment:
```suggestion
    from django.conf import settings
    from django import VERSION as DJANGO_VERSION
```

We may need to import here or at the top of the file.